static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-load.js

```diff
@@ -43,25 +48,59 @@ const build = (
     return html;
 };
 
-const load = (target?: string): Promise<void> => {
-    const slot = target && target in selectors ? target : 'defaults';
+/*
+  Load the outbrain widget according to target and to the visibility of
+  the contribution epic
+
+  Valid values for target are 'defaults', 'merchandising', 'nonCompliant'
+  
+  When the target is nonCompliant, you are encouraged to pass the contributionEpicVisible
+  parameter as this will have an effect on the new outbrain ID mapping (as of end 2018).
+
+ */
+const load = (
+    target: string,
+    contributionEpicVisible?: boolean = false
+): Promise<void> => {
+    const outbrainType = target && target in selectors ? target : 'defaults';
     const $outbrain = $(selectors.outbrain.widget);
     const $container = $(selectors.outbrain.container, $outbrain[0]);
     const breakpoint = getBreakpoint();
 
-    const widgetCodes = getCode({
-        slot,
-        section: config.page.section,
-        breakpoint,
-    });
+    const useNewOutbrainCodes =
+        getTestVariantId('CommercialOutbrainNewids') === 'variant';
+
+    const widgetCodes = ((): {
+        code?: string,
+        image?: string,
+        text?: string,
+    } => {
+        if (useNewOutbrainCodes) {
+            return getNewCode({
+                outbrainType,
+                contributionEpicVisible,
+                section: config.page.section,
```

Comment:
We have a convention of `no-direct-access-config` therefore `config.page.section` could be replaced with a use of the `get()` funtion from the config module:

> [config] should be the only module accessing the window config object directly

> [get] allows you to safely get items from config using a query of dot or bracket notation, with optional default fallback

```js
config.get('page.section')
```